static/app/components/feedback/useFetchFeedbackInfiniteListData.tsx

```diff
@@ -0,0 +1,305 @@
+import {useCallback, useEffect, useRef, useState} from 'react';
+import {Index, IndexRange} from 'react-virtualized';
+import moment from 'moment';
+
+import {ApiResult, Client} from 'sentry/api';
+import hydrateFeedbackRecord from 'sentry/components/feedback/hydrateFeedbackRecord';
+import {Organization} from 'sentry/types';
+import formatDuration from 'sentry/utils/duration/formatDuration';
+import Dispatch from 'sentry/utils/eventDispatcher';
+import {
+  FeedbackItemResponse,
+  HydratedFeedbackItem,
+} from 'sentry/utils/feedback/item/types';
+import {EMPTY_QUERY_VIEW, QueryView} from 'sentry/utils/feedback/list/types';
+import parseLinkHeader from 'sentry/utils/parseLinkHeader';
+import {decodeInteger} from 'sentry/utils/queryString';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+
+const PER_PAGE = 10;
+
+type Unsubscribe = () => void;
+
+function startDateFromQueryView({start, statsPeriod}: QueryView): Date {
+  if (start) {
+    return new Date(start);
+  }
+  if (statsPeriod) {
+    const value = parseInt(statsPeriod, 10);
+    const unit = statsPeriod.endsWith('m')
+      ? 'min'
+      : statsPeriod.endsWith('h')
+      ? 'hour'
+      : statsPeriod.endsWith('d')
+      ? 'day'
+      : statsPeriod.endsWith('w')
+      ? 'week'
+      : undefined;
+    if (unit) {
+      const msdifference = formatDuration({
+        precision: 'ms',
+        style: 'count',
+        timespan: [value, unit],
```

Comment:
off topic: "timespan" implies to me that it has at least two points of time vs what it is now (single point)

Reply:
i can see that makes sense yeah. 

it's called formatDuration, so it should just be `duration:`, another cromulent alternative might be `interval` `timeInterval`, `formatInterval()` ?